ecs-agent/netlib/platform/api.go

```diff
@@ -33,6 +33,9 @@ type API interface {
 		taskID string,
 		taskPayload *ecsacs.Task) (*tasknetworkconfig.TaskNetworkConfig, error)
 
+	// HandleHostMode returns error if host mode is not enabled for the platform
```

Comment:
nit: end comments with `.`

Reply:
Same comment on L91 of `ecs-agent/netlib/platform/common.go`

Reply:
done